docs-site/src/markdown-pages/technical-design/index.md

```diff
@@ -59,14 +60,15 @@ This allows councils to control the frequency of updates. However, there will be
 some cases where pulling data from a system is required, such as data from third
 party systems that can't be changed.
 
-A special case is Hilltop. Because of its wide-use and existing APIs, EOP will
-support pulling data from council Hilltop instances.
+There will be special cases for tools that are ubiquitous across the councils
+that EOP can pull from standard interfaces without per council integration. For
+example, ESRI and Hilltop.
```

Comment:
This is the bit that is an attempt at a re-word rather than just fixing errors.

Jason raised that calling out hilltop seemed odd ... which I agree with, and I think the correct wording is that EOP will custom pull/integrate with systems that are common amongst councils